packages/teams-js/src/public/runtime.ts

```diff
@@ -437,7 +437,7 @@ export const mapTeamsVersionToSupportedCapabilities: Record<string, Array<ICapab
   '2.0.5': [
     {
       capability: { webStorage: {} },
-      hostClientTypes: [HostClientType.android, HostClientType.desktop, HostClientType.ios],
```

Comment:
>HostClientType.desktop [](http://example.com/codeflow?start=48&length=22)

I suspect that `HostClientType.desktop` was added here to workaround the bug I'm fixing in these tests.

This map should only represent the version when a capability first started working on a platform; that platform doesn't need to then be added to all newer versions. Support going forward is presumed.